### ISSUE TITLE
How do I use with byte[] Images?

### ISSUE THREAD

**Issue by nathan1658** [stormlion227/PhotoBrowser.Forms#11]

**Comment by vniehues** [comment]
bump

**Comment by LeoJHarris** [comment]
I don't think this is a simple thing to do as the Xamarin Forms Library is reliant on the native library implementations for both iOS and Android, if this were to be integrated it would likely need to be made in those native binding libraries. It could be done but it probably wouldn't be a simple change.

**Comment by gurneetsinghb** [comment]
any solution for this issue?
